packages/web/src/scss/components/Navigation/README.md

```diff
@@ -0,0 +1,82 @@
+# Navigation
+
+The `Navigation` component is a container for the navigation links of the application.
+
+It consists of a these parts:
+
+- [Navigation](#navigation)
+- [NavigationLink](#navigationlink)
+
+## Navigation
+
+The `Navigation` is a `nav` wrapper for lists of links or other navigation components.
+
+```html
+<nav class="Navigation">
+  <ul>
+    <li>
+      <!-- link -->
+    </li>
+  </ul>
+</nav>
+```
+
+It centers its children vertically and if the children are not NavigationLink components,
+it will apply a gap between them.
+
+## NavigationLink
+
+The `NavigationLink` is component that is styled to be used as a navigation link.
+
+```html
+<a class="NavigationLink" href="#">Link</a>
+```
+
+If can obtain `selected` or `disabled` states by adding the respective classes.
+
+```html
+<a class="NavigationLink NavigationLink--selected" href="#" aria-current="page">Selected Link</a>
+<a class="NavigationLink NavigationLink--disabled" href="#">Disabled Link</a>
```

Comment:
- The "disabled" link should be a `<span>` to be semantically correct (or not to be incorrect) and to prevent it from user focus.
- Do we need disabled navigation links at all?

![image](https://github.com/user-attachments/assets/bfbd777f-e238-40fc-b155-fa933f4e6272)

Reply:
It is in design, so I prepared it. I switched to span and added it to the documentation.